ml-conversational-analytic-tool/githubDataExtraction.py

```diff
@@ -158,20 +152,28 @@ def getPullFeatures(self, pull):
 
 
 if __name__ == "__main__":
-    parser = argparse.ArgumentParser(description='Create csv for all pulls in repo')
+    parser = argparse.ArgumentParser(description='Create CSV/s for all pulls in repo/s')
     parser.add_argument('organization', help='Organization the repo belongs to.')
-    parser.add_argument('reponame', help='Name of repo')
-    parser.add_argument('-reactions', action='store_true', default=False, help='Flag to extract reactions')
-    parser.add_argument('--filename', help='Name of file')
-    # parser.add_argument('accesstoken', help='Github access token')
+    parser.add_argument('-R, --repo', help='Name of repo.')
```

Comment:
small but important nil:
Notice `'-R, --repo'` should not be a single string, but `'-R', '--repo'`

Correct: `parser.add_argument('-R', '--repo', help='Name of repo.')`

Reply:
Thanks